standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/DatabaseProduct.java

```diff
@@ -263,7 +265,9 @@ protected String toDate(String tableValue) {
 
   protected String toTimestamp(String tableValue) {
     if (isORACLE()) {
-      return "TO_TIMESTAMP(" + tableValue + ", 'YYYY-MM-DD HH:mm:ss')";
+      return "TO_TIMESTAMP(" + tableValue + ", 'YYYY-MM-DD HH:mi:ss')";
```

Comment:
Timezone information does not need here?

Reply:
I haven't tested against Oracle yet, let me try

Reply:
We don't need it here, I have tested against Oracle19